classes/install/Upgrade.inc.php

```diff
@@ -2430,7 +2431,22 @@ function moveCSSFiles() {
 		return true;
 	}
 
+	/**
+	 * For 3.1.0 upgrade.  DefaultMenus Defaults
+	 */
+	function installDefaultNavigationMenus() {
+		$journalDao = DAORegistry::getDAO('JournalDAO');
+		$navigationMenuDao = DAORegistry::getDAO('NavigationMenuDAO');
 
+		$journals = $journalDao->getAll();
+		while ($journal = $journals->next()) {
+			$navigationMenuDao->installSettings($journal->getId(), 'registry/navigationMenus.xml');
+		}
+
+		$navigationMenuDao->installSettings(CONTEXT_ID_NONE, 'registry/navigationMenus.xml');
+
+		return true;
+	}
```

Comment:
Should these changes be moved to `Installer.inc.php` in `pkp-lib`, and `JournalDAO` references be changed for `ContextDAO`?

Reply:
Changed to ContextDAO.
I think that the code should be on Upgrade process as well.